docs/en_US/Tutorial/SetupNniDeveloperEnvironment.md

```diff
@@ -48,7 +48,8 @@ Nothing to do, the code is already linked to package folders.
 #### TypeScript
 
 * If `src/nni_manager` is changed, run `yarn watch` under this folder. It will watch and build code continually. The `nnictl` need to be restarted to reload NNI manager.
-* If `src/webui` or `src/nasui` are changed, run `yarn start` under the corresponding folder. The web UI will refresh automatically if code is changed.
+* If `src/webui` is changed, run `yarn dev`, which will run a mock API server and a webpack dev server simultaneously. Use `EXPERIMENT` environment variable (e.g., `mnist-tfv1-running`) to change mock data. Built-in mock experiments are listed in `src/webui/mock`.

```

Comment:
better to provide the entire command such as `EXPERIMENT=mnist-tfv1-running yarn dev`